generator/generator_config.textproto

```diff
@@ -3268,7 +3268,6 @@ service {
 service {
   service_proto_path: "google/cloud/telcoautomation/v1/telcoautomation.proto"
   product_path: "google/cloud/telcoautomation/v1"
-  forwarding_product_path: "google/cloud/telcoautomation"
```

Comment:
comment: I assume that if the unversioned types were in the ABI baseline, `check-api-pr` would have failed.

Reply:
I doubt it.  I think all the unversioned types are not types, just type aliases.  These are not part of the ABI and therefore invisible to the `check-api-pr` tooling.

Reply:
Oh.... So if we removed a `using Foo = ...` from our public API, the build would not catch it... That is unfortunate.